shells/fish/Portfile

```diff
@@ -5,11 +5,11 @@ PortGroup               github 1.0
 PortGroup               cmake 1.1
 PortGroup               legacysupport 1.0
 
-github.setup            fish-shell fish-shell 3.6.1
+github.setup            fish-shell fish-shell 3.6.3
 revision                1
```

Comment:
revision field should be reset to zero when updating the version.

Reply:
sorry @reneeotten for some reason this always throws me off.